src/components/Multiselect/AvatarSelectOption.vue

```diff
@@ -30,7 +30,8 @@ export default {
 					desc: '',
 					displayName: 'Admin',
 					icon: 'icon-user',
-					user: 'admin'
+					user: 'admin',
+					isNoUser: false
```

Comment:
does it make sense to rename to `isGroup`?

Reply:
@ChristophWurst bump?

Reply:
Fine by me :+1: